user/build-stages.md

```diff
@@ -9,45 +9,79 @@ layout: en
 > Build stages are currently in BETA. There is more information about what this means, and how you can give us feedback on this new feature in the [GitHub issue](https://github.com/travis-ci/beta-features/issues/11).
 {: .beta}
 
-With this new feature you can group jobs together in 'stages'. Jobs in a stage
-run in parallel, and the stages themselves run sequentially, one after another.
-The build fails when any stage fails, i.e. if any job in this stage fails.
+# What are Build Stages?
 
-For example, you can configure a deployment stage to run only after several test
-jobs have all completed successfully.
+In the simplest and most common use case, you can now make one job run _only_
```

Comment:
Maybe something like

Build stages is a way to group jobs, run them in parallel, and then run other jobs depending on the overall return status of the group.